content/docs/pipelines/deployment-alternatives.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "Pipelines Standalone Deployment Alternatives"
+description = "Alternative approaches to deploy Kubeflow Pipelines standalone to a cluster"
+weight = 15
++++
+
+As an alternative to [deploying Kubeflow](/docs/started/getting-started/#installing-kubeflow) as a
+whole with many components including pipelines, you also have a choice to deploy
+only Kubeflow Pipelines. Follow the instructions below to deploy
+Kubeflow Pipelines standalone using the supplied kustomize
+manifests.
+
+Knowledge about [Kubernetes](https://kubernetes.io/docs/home/), [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) and [kustomize](https://kustomize.io/) will help you understand this
+document better and be able to customize based on your needs.
+
+
+## Deploying Kubeflow Pipelines standalone to an existing cluster
+
+1. Deploy latest version of Kubeflow Pipelines:
+```
+export PIPELINE_VERSION={{% kfp-latest-version %}}
+kubectl apply -k github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
+```
+
+2. Get the URL for the Kubeflow Pipelines UI :
+```
+kubectl describe configmap inverse-proxy-config -n kubeflow | grep googleusercontent.com 
+```
+
+## Deploying Kubeflow Pipelines standalone from scratch
+
+1. Prepare a Kubernetes cluster:
+
+    You have many options from cloud service providers. For Google Cloud
+    Platform (GCP), you can reference [Creating a cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-cluster).
+
+    You can also set up your own Kubernetes cluster.
+
+    Cluster minimum requirement: node pool with at least 2 nodes and 2 CPUs per node.
```

Comment:
do you know if this is the minimum requirements for non gcp machines?

Reply:
I don't know, does it work for non gcp since we use inverse-proxy?

Reply:
Tested n1-standard-2 with 2 nodes can run all samples.